android/src/main/java/com/swmansion/rnscreens/ScreenFragment.kt

```diff
@@ -74,13 +74,12 @@ open class ScreenFragment : Fragment {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        val wrapper = context?.let { ScreensFrameLayout(it) }
-
-        val params = FrameLayout.LayoutParams(
-            ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT
-        )
-        screen.layoutParams = params
-        wrapper?.addView(recycleView(screen))
+        val wrapper = context?.let { ScreensFrameLayout(it) }?.apply {
```

Comment:
It introduces change of behavior since the operations inside `apply` won't be done if `wrapper` is not there. I am not sure if it should be this way, but it would be nice to check if `wrapper` is ever not found.

Reply:
Good catch: [c370c24](https://github.com/software-mansion/react-native-screens/pull/1558/commits/c370c242e4c0949459f5554ee82170b6beeb9d3b).